src/FindSegmentsMsg.chpl

```diff
@@ -37,20 +37,13 @@ module FindSegmentsMsg
         var repMsg: string; // response message
         // split request into fields
         var (pname, nkeysStr, rest) = payload.splitMsgToTuple(3);
```

Comment:
we don't need the `splitMsgToTuple` line anymore

Reply:
Removed. I believe we could probably get rid of this file though... Maybe?

Reply:
yep! it looks like this used to be used by `GroupBy` but `findSegments` was removed in https://github.com/Bears-R-Us/arkouda/pull/1330. I think it should be safe to remove

Reply:
I also built the server with `FindSegmentsMsg` commented out in `ServerModules.cfg` and verified all the tests pass

Reply:
I will remove the file and push that to this PR.

Reply:
Updated this PR to remove the File